Sources/Models/Views/ScrollViews/CollectionView.swift

```diff
@@ -35,6 +35,7 @@ public struct CollectionView: IBDecodable, ViewProtocol, IBIdentifiable {
     public let connections: [AnyConnection]?
     public let variations: [Variation]?
     public let cells: [CollectionViewCell]?
+    public let collectionReusableView: [CollectionReusableView]?
```

Comment:
ok you make the choice to decode as a list

Reply:
if we want direct accessor we can make it with code
```swift
var sectionHeaderView: CollectionReusableView? {
   return collectionReusableView?.filter { $0.key == "sectionHeaderView" }
}
```
(not tested)

Reply:
yes, I’ll do it if you want.

Reply:
yes if you can 😇

Reply:
alright, did it.
and rename collectionReusableView to collectionReusableView`s`.